lib/puppet/indirector/catalog/compiler.rb

```diff
@@ -123,6 +124,17 @@ def inlineable?(resource, sources)
     end
   end
 
+  # Return true if metadata is inlineable, meaning the request's source is
+  # for the 'modules' mount and the resolved path is of the form:
+  #   $codedir/environments/$environment/*/*/files/**
+  def inlineable_metadata?(metadata, source, environment_path)
+    source_as_uri = URI(URI.escape(source))
+    location = Puppet::Module::FILETYPES['files']
+
```

Comment:
It's not strictly necessary, but since it's a predicate method, it's ruby convention to double negate to ensure a boolean value is returned. A truthy or falsey value would also work, but it leaks internal state.